nodes/index.adoc

```diff
@@ -129,3 +129,79 @@ As an administrator, You can perform various tasks on a Linux container, such as
 {product-title} provides specialized containers called xref:../nodes/containers/nodes-containers-init.adoc#nodes-containers-init[Init containers]. Init containers run before application containers and can contain utilities or setup scripts not present in an application image. You can use an Init container to perform tasks before the rest of a pod is deployed.
 
 Apart from performing specific tasks on nodes, pods, and containers, you can work with the overall {product-title} cluster to keep the cluster efficient and the application pods highly available.
+
+[id="commonterms-node"]
+== Common terms
+
+This glossary defines common terms that are used in the _node_ content. These terms help you understand _nodes_ effectively.
+
+[discrete]
+[id="commonterms-node-container"]
+Container::
+It is a lightweight and executable image that comprises software and all its dependencies. Containers virtualize the operating system, as a result, you can run containers anywhere from a data center to a public or private cloud to even a developer's laptop.
+
+[discrete]
+[id="commonterms-node-daemonset"]
+Daemon set::
+Ensures that a replica of the pod runs on eligible nodes in an {product-title} cluster.
+
+[discrete]
+[id="commonterms-node-egress"]
+egress::
+The process of data sharing externally through a network’s outbound traffic from a pod.
+
+[discrete]
+[id="commonterms-node-gc"]
+garbage collection::
+The process of cleaning up cluster resources, such as terminated containers and images that are not referenced by any running pods.
+
+[discrete]
+[id="commonterms-node-hpa"]
+Horizontal Pod Autoscaler(HPA)::
+Implemented as a Kubernetes API resource and a controller. You can use the HPA to specify the minimum and maximum number of pods that you want to run. You can also specify the CPU or memory utilization that your pods should target. The HPA scales out and scales in pods when a given CPU or memory threshold is crossed. 
+
+[discrete]
+[id="commonterms-node-ingress"]
+Ingress::
+Is incoming traffic to a pod.
```

Comment:
Looks like this one was unaddressed. Did you want to address it? 

"Is incoming traffic to a pod" is an incomplete sentence.

Reply:
addressed